docs/reference/api/serving.md

```diff
@@ -24,7 +24,7 @@ Resource Types:
 <p>PodAutoscaler is a Knative abstraction that encapsulates the interface by which Knative
 components instantiate autoscalers.  This definition is an abstraction that may be backed
 by multiple definitions.  For more information, see the Knative Pluggability presentation:
-<a href="https://docs.google.com/presentation/d/e/2PACX-1vQ4kT92QNXvBF9dJzVfF_zqUROoG1wZXf-ATMKi8d43htvt3Osq6k-1YUWAO9M27gZ8C-cGBQA8-bwJ/pub">https://docs.google.com/presentation/d/e/2PACX-1vQ4kT92QNXvBF9dJzVfF_zqUROoG1wZXf-ATMKi8d43htvt3Osq6k-1YUWAO9M27gZ8C-cGBQA8-bwJ/pub</a></p>
+<a href="https://docs.google.com/presentation/d/10KWynvAJYuOEWy69VBa6bHJVCqIsz1TNdEKosNvcpPY/edit">https://docs.google.com/presentation/d/10KWynvAJYuOEWy69VBa6bHJVCqIsz1TNdEKosNvcpPY/edit</a></p>
```

Comment:
I'd change this to something like:

"For more information, see the <a href="https://docs.google.com/presentation/d/10KWynvAJYuOEWy69VBa6bHJVCqIsz1TNdEKosNvcpPY/edit">Knative pluggability presentation</a></p>."

Reply:
Similar thing here too: https://5ffe5166b7462545d9c8fc4d--knative.netlify.app/docs/reference/api/serving-api/#serving.knative.dev/v1.Configuration
I'd try to show descriptive names for all the links rather than:
"See also: https://github.com/knative/serving/blob/master/docs/spec/overview.md#configuration"

Reply:
I think most sections of this preview have similar issues

Reply:
@abrennan89 all these strings are located in the source code, we only just build and publish an HTML version. I do agree that these are improvements but the edits should be made in the source rather than here in the output. Let me know if thats something you want to dive into. Alternatively, open a bug for the Serving team to get this fixed for the next release?

Reply:
Makes sense, opened an issue to track it for the next release :slightly_smiling_face: